### DISCUSSION TITLE
lernos-zettelkasten/de/1-2-lernOS-Canvas/

### DISCUSSION THREAD

**Discussion by giscus** [cogneon/lernos-zettelkasten#11] [Announcements]
# lernos-zettelkasten/de/1-2-lernOS-Canvas/

https://cogneon.github.io/lernos-zettelkasten/de/1-2-lernOS-Canvas/

**Comment by karstenpe** [comment]
>> Alex Osterwalder hatte als erster die Idee für einen lernOS Canvas.
Eher nicht, oder?
Er ist der geistige Vater des Business Model Canvas.
lernOS ist von Simon/cogneon und kam später.
Grüße,
Karsten

**Reply by saertna** [reply]
Danke für den Hinweis @karstenpe, ich nehme das mit zur Überarbeitung.
